src/utils/db.py

```diff
@@ -21,6 +22,19 @@ def get_session() -> Session:
     engine = get_engine()
     return Session(engine)
 
+def add_object_list(obj_list: List[Base]) -> dict | str:
```

Comment:
Pretty sure you can do `obj_list: list[Base]`, and then you don't need the `typing` import. I believe the way you've done it here was the old way, before type hinting was officially implemented into Python.

Reply:
Oh interesting, I had no idea. I just saw that syntax on a stack overflow somewhere and copied it. Will try with list instead